code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -1109,7 +1108,7 @@
 
 /obj/item/reagent_containers/food/snacks/enchiladas
 	name = "enchiladas"
-	desc = "Viva La Mexico!"
+	desc = "Rolled meat chili and corn into one."
```

Comment:
"Meat, chili and corn rolled into a tortilla"

or "Meat, chili and corn rolled in one tortilla."